src/ResourceManager/Network/Commands.Network/help/Set-AzureRmVirtualNetworkGateway.md

```diff
@@ -13,14 +13,26 @@ Updates a virtual network gateway.
 
 ## SYNTAX
 
+### Empty (Default)
 ```
 Set-AzureRmVirtualNetworkGateway -VirtualNetworkGateway <PSVirtualNetworkGateway> [-GatewaySku <String>]
  [-GatewayDefaultSite <PSLocalNetworkGateway>]
  [-VpnClientAddressPool <System.Collections.Generic.List`1[System.String]>]
  [-VpnClientRootCertificates <System.Collections.Generic.List`1[Microsoft.Azure.Commands.Network.Models.PSVpnClientRootCertificate]>]
  [-VpnClientRevokedCertificates <System.Collections.Generic.List`1[Microsoft.Azure.Commands.Network.Models.PSVpnClientRevokedCertificate]>]
  [-Asn <UInt32>] [-PeerWeight <Int32>] [-EnableActiveActiveFeature] [-DisableActiveActiveFeature]
- [<CommonParameters>]
+ [-RadiusServerSecret <SecureString>] [<CommonParameters>]
```

Comment:
Is this intentional?  Why would radius server secret show up here (commented in the code as well)

Reply:
This is auto generated - I do not know why it is showing up here